doc/hardware/porting/board_porting.rst

```diff
@@ -650,6 +650,9 @@ while porting.
 - If available, configure and enable a serial output for the console
   using the ``zephyr,console`` chosen node in the devicetree.
 
+- If available, configure and enable a hardware entropy generator using the
+  ``zephyr,entropy`` chosen node in the devicetree.
```

Comment:
Thanks. 
Now, the remaining part of https://github.com/zephyrproject-rtos/zephyr/issues/82344 fix will have to explain what to do when this is not the case (to avoid CI failures seen currently)